src/libraries/Microsoft.Extensions.DependencyInjection/tests/DI.External.Tests/Unity.cs

```diff
@@ -7,6 +7,9 @@ namespace Microsoft.Extensions.DependencyInjection.Specification
 {
     public class UnityDependencyInjectionSpecificationTests: SkippableDependencyInjectionSpecificationTests
     {
+        // See https://github.com/unitycontainer/microsoft-dependency-injection/issues/87
+        public override bool ExpectStructWithPublicDefaultConstructorInvoked => true;
```

Comment:
You can use the `SkippedTests` for this.

Reply:
I didn't want to skip the whole test, I just wanted to change this one assert. It didn't seem important enough to add a whole new test. And, in my opinion, it feels more correct to verify the behavior, even if the behavior is not "correct".

Reply:
As long as 3rd party library authors understand that this is something they are expected to fix.

Reply:
We might want to cc all of them on this PR. @davidfowl had a list somewhere.

Reply:
Unity was the one with this behavior. I opened the above issue (which points back to this PR) to start the discussion on if this is the expected behavior in Unity or not.

Reply:
based on https://github.com/dotnet/runtime/issues/35999 the alias for unity is @ENikS